spec/turtle.bnf

```diff
@@ -23,10 +23,10 @@ String                ::= STRING_LITERAL_QUOTE | STRING_LITERAL_SINGLE_QUOTE
 iri                   ::= IRIREF | PrefixedName
 PrefixedName          ::= PNAME_LN | PNAME_NS
 BlankNode             ::= BLANK_NODE_LABEL | ANON
-quotedTriple          ::= '<<' qtSubject predicate qtObject '>>'
-qtSubject             ::=	iri | BlankNode | quotedTriple
-qtObject              ::=	iri | BlankNode | literal | quotedTriple
-annotation            ::= '{|' predicateObjectList '|}'
+reifier               ::= '<<' ((iri | BlankNode) '|' )? subject predicate object '>>'
+tripleTerm            ::= '<<(' subject predicate ttObject ')>>'
+ttObject              ::=	iri | BlankNode | literal | tripleTerm
+annotation            ::= '{|' ( (iri | BlankNode) '|' )? predicateObjectList '|}'
```

Comment:
Based on recently seen examples that deal with the ambiguity of using `|` to separate the reifier from the predicateObjectList, we could consider using `=>` instead.

```suggestion
annotation            ::= '{|' ( (iri | BlankNode) '=>' )? predicateObjectList '|}'
```

This would also apply to line 26 (where we might want to change the name of the `reifier` production to something like `reifiedTripleTerm`, as `reifier` is used elsewhere to describe `(ire | BlankNode)` in that production.